modules/obscalc/src/main/scala/lucuma/odb/obscalc/Main.scala

```diff
@@ -227,12 +233,14 @@ object CalcMain extends MainParams:
       pool  <- databasePoolResource[F](c.database)
       enums <- Resource.eval(pool.use(Enums.load))
 
+      http  <- c.httpClientResource
       itc   <- c.itcClient
       ptc   <- Resource.eval(pool.use(TimeEstimateCalculatorImplementation.fromSession(_, enums)))
 
       t     <- topic(pool)
       user  <- Resource.eval(serviceUser[F](c))
-      o     <- runObscalcDaemon(c.database.maxObscalcConnections, c.commitHash, c.obscalcPoll, itc, ptc, t, pool.evalMap(services(user, enums)))
+      mapping = OdbMapping.forObscalc(pool, SkunkMonitor.noopMonitor[F], user, c.commitHash, enums, http)
```

Comment:
We need a GraphQL mapping since `Obscalc` now computes the observation workflow (which does a nested GraphQL call).  For that reason, the service user code is shuffled around a bit to make a user available here to pass to the mapping.